src/w3d/renderer/dx8renderer.cpp

```diff
@@ -724,17 +724,26 @@ unsigned int DX8TextureCategoryClass::Add_Mesh(Vertex_Split_Table &split_table,
                     if (Compare_Materials(split_table.Peek_Material(i, pass), m_material)) {
                         if (split_table.Peek_Shader(i, pass) == m_shader) {
                             unsigned short index = vertex_offset + polys[i].I;
-                            std::clamp(index, vmin, vmax);
+
```

Comment:
Just a thought but couldn't this be turned into a for loop?
Not sure if that makes it more readable.
```
for (auto j = 0; j < 3; ++j){
    uint16_t index = vertex_offset + polys[i][j];
    vmin = std::min(index, vmin);
    vmax = std::max(index, vmax);
    *indices++ = index;
}
```